es/ae/oracle.js

```diff
@@ -190,14 +182,15 @@ async function postQueryToOracle (oracleId, query, options = {}) {
   const opt = R.merge(this.Ae.defaults, options)
```

Comment:
Also here. Can the usage of R be replaced by `const opt = { ...this.Ae.defaults, ...options }`